test/golden/Inline2.err

```diff
@@ -0,0 +1,2 @@
+test/Fail/Inline2.agda:5,1-6
+Cannot make function tail' inlinable. Inline functions can only use variable patterns, dot patterns, or transparent record constructor patterns.
```

Comment:
Very minor comment: you cannot have dot patterns without having at least one constructor pattern with a constructor of an indexed datatype. So the mention of dot patterns here is redundant.